frontend/src/components/UsersList.js

```diff
@@ -20,9 +23,22 @@ const UsersList = () => {
             </tr>
           </thead>
           <tbody>
-            {users?.map((user) => (
-              <ListElement user={user} key={user.id} />
-            ))}
+            {users
+              ?.filter((user) =>
```

Comment:
A filterezést lehet be lehetne rejteni a useUsers hookon belül, például useUsers(searchKeyword). Ez úgy nagyon clean lenne itt és a komplexitást meg a hookon belül kezelhetnénk.